src/Microsoft.DotNet.Cli.Utils/Product.cs

```diff
@@ -1,22 +1,17 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.Reflection;
-
 namespace Microsoft.DotNet.Cli.Utils
 {
     public class Product
     {
-        public static string LongName => LocalizableStrings.DotNetCommandLineTools;
+        public static string LongName => LocalizableStrings.DotNetSdkInfo;
         public static readonly string Version = GetProductVersion();
 
         private static string GetProductVersion()
         {
-            var attr = typeof(Product)
-                .GetTypeInfo()
-                .Assembly
-                .GetCustomAttribute<AssemblyInformationalVersionAttribute>();
-            return attr?.InformationalVersion;
+            DotnetVersionFile versionFile = DotnetFiles.VersionFileObject;
+            return versionFile.BuildNumber ?? string.Empty;
```

Comment:
Just confirm that we decide to simply return empty in this case

Reply:
When John and I talked last, I thought we were going to make it a hard failure with the reasoning that the version file can be considered a required part of the installation.

Reply:
I'm OK with empty and Will makes a good point about not requiring the core-sdk bundling to execute dotnet.dll. We have a lot of related problems though, so at this point in time, I could go either way.